draft-ietf-taps-impl.md

```diff
@@ -533,53 +532,53 @@ The effect of the application sending a Message is determined by the top-level p
 
-- `final`: when this is true, this means that the sender will not send any further messages. The Connection need not be closed (in case the Protocol Stack supports half-close operation, like TCP). Any messages sent after a Final message will result in a SendError.
+- `final`: when this is true, this means that the sender will not send any further Messages. The Connection need not be closed (in case the Protocol Stack supports half-close operation, like TCP). Any Messages sent after a final Message will result in a SendError.
 
 - `msgChecksumLen`: when this is set to any value other than `Full Coverage`, it sets the minimum protection in protocols that allow limiting the checksum length (e.g. UDP-Lite). If the Protocol Stack does not support checksum length limitation, this property may be ignored.
 
 - `msgReliable`: When true, the property specifies that the Message must be reliably transmitted. When false, and if unreliable transmission is supported by the underlying protocol, then the Message should be unreliably transmitted. If the underlying
 protocol does not support unreliable transmission, the Message should be reliably transmitted.
 
 - `msgCapacityProfile`: When true, this expresses a wish to override the
-Generic Connection Property `Capacity Profile` for this Message. Depending on the
+Generic Connection Property `connCapacityProfile` for this Message. Depending on the
 value, this can, for example, be implemented by changing the DSCP value of the
 associated packet (note that the guidelines in Section 6 of {{?RFC7657}} apply; e.g.,
 the DSCP value should not be changed for different packets within a reliable
 transport protocol session or DCCP connection).
 
-- `noFragmentation`: Setting this avoids network-layer fragmentation. Messages exceeding the transport’s current estimate of its maximum packet size (the `singularTransmissionMsgMaxLen` Connection Property) can result in transport segmentation when permitted, or generate an error. When used with transports running over IP version 4 the Don't Fragment bit will be set to avoid on-path IP fragmentation ({{!RFC8304}}).
+- `noFragmentation`: Setting this avoids network-layer fragmentation. Messages exceeding the transport’s current estimate of its maximum packet size (the `singularTransmissionMsgMaxLen` Connection Property) can result in transport segmentation when permitted, or generate an error. When used with transports running over IP version 4, the Don't Fragment bit should be set to avoid on-path IP fragmentation ({{!RFC8304}}).
```

Comment:
This is a guide for developers, not a list of facts about how existing systems behave - that was my rationale here.